client/lib/plans/constants.js

```diff
@@ -107,9 +107,11 @@ export const plansList = {
 		getDescription: () => i18n.translate( 'Your own domain name, powerful customization options, and' +
 			' lots of space for audio and video.' ),
 		getDescriptionWithGoogleVouchers: () => i18n.translate( 'Your own domain name, powerful customization options,' +
-			' lots of space for audio and video, and $100 advertising credit.' ),
+			' lots of space for audio and video, and $100 advertising voucher.' ),
 		getDescriptionWithWordAdsInstantActivation: () => i18n.translate( 'Your own domain name, powerful' +
 			' customization options, easy monetization with WordAds and lots of space for audio and video.' ),
+		getDescriptionWithWordAdsAndGoogleVouchers: () => i18n.translate( 'Your own domain name, powerful' +
```

Comment:
Sure, you're right. The verbosity freaked me out, but it is more clear. These will be somewhat short-lived anyway.